wordpress-mobile/WordPress-Android
# Feature/use formattable content in activity log

Fixes #8276
I'm adding a new Click listener that extract click handling on formattable content. This listener is being passed into NotificationUtilsWrapper. NotificationsUtilsWrapper is called to create a Spannable from Formattable content in Activity Log. As a result we can now see formattable content with links in Activity Log.  

![formattable_content_in_activity_log](https://user-images.githubusercontent.com/1079756/45162428-4a332a00-b1ee-11e8-81ed-9b6eedf6ff4b.gif)

To test:
- Go to activity log
- Click on items that link to content (for example mentioning a Site, a Link or a Comment)
- You see that the text of the Content is now blue and clickable
- Clicking on the content opens it in a new window
